src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/PlainText.txt

```diff
@@ -0,0 +1,97 @@
+>> PlainText("")
```

Comment:
>PlainText [](http://example.com/codeflow?start=3&length=9)

Just double checking: these are all correct for PowerApps ? This .txt will run against JS tests too.

Reply:
all these tests have the same results with PA.